code/perception/src/radar_node.py

```
@@ -5,21 +5,102 @@
 import json
 from sensor_msgs import point_cloud2
 from visualization_msgs.msg import Marker, MarkerArray
-from tf.transformations import quaternion_from_matrix
 from std_msgs.msg import Float32MultiArray
+from tf.transformations import quaternion_from_matrix
+
 import struct
+import numpy as np
+from math import radians, cos, sin
+from ros_compatibility.node import CompatibleNode
+import sensor_msgs.point_cloud2 as pc2
```

Ruff (0.8.2)

`sensor_msgs.point_cloud2` imported but unused

Remove unused import: `sensor_msgs.point_cloud2`

(F401)